### DISCUSSION TITLE
挂载/订阅网盘别人公开分享的链接

### DISCUSSION THREAD

**Discussion by BingoKingo** [AlistGo/alist#5207] [Q&A]
如题，想挂载阿里云盘、百度云盘、123网盘、MEGA等网盘别人分享出来的文件夹，订阅内容更新，方便自动获取新内容直链，不用打开官方客户端检查更新

目前尝试百度云盘分享挂载需会员，阿里云盘分享token提示错误...

**Comment by kaixinxiaowangzi666** [comment]
早就不能挂载分享连接了
